proposals/168/making_spark_visitors_extensible.md

```diff
@@ -259,10 +259,10 @@ public class OutputStatisticsFacetBuilder implements CustomFacetBuilder<StageInf
     @Override
     public void accept(StageInfo event, BiConsumer<String, ? super OpenLineage.OutputStatisticsOutputDatasetFacet> consumer) {
         OpenLineage.OutputStatisticsOutputDatasetFacet statsFacet =
-                openLineage.newOutputStatisticsOutputDatasetFacet(
-                        event.taskMetrics().outputMetrics().recordsWritten(),
-                        event.taskMetrics().outputMetrics().bytesWritten()
-                );
+                openLineage.newOutputStatisticsOutputDatasetFacetBuilder()
+                  .rowCount(event.taskMetrics().outputMetrics().recordsWritten())
+                  .size(event.taskMetrics().outputMetrics().bytesWritten())
+                  .build();
```

Comment:
I don't think we need to update proposals :)

Reply:
Oops, sorry)